src/Activation/ActivateProximityTracing.tsx

```diff
@@ -13,7 +13,7 @@ import { useNavigation } from "@react-navigation/native"
 
 import { usePermissionsContext } from "../PermissionsContext"
 import { useOnboardingContext } from "../OnboardingContext"
-import { useApplicationName } from "../More/useApplicationInfo"
+import { useApplicationName } from "../hooks/useApplicationInfo"
```

Comment:
So far we have been avoiding `directory by implementation` and preferring `directory by feature`. What do you think about keeping this convention the same? Is there a benefit to grouping all the hooks together, especially if a hook is only used in one place and is not intended to be used throughout the app?

Reply:
I agree that there's no benefit to grouping all the hooks together, and this PR does not move all of the hooks into this directory. This specific hook is currently very widely used throughout the app (Home, Activation, Onboarding, More, ReportIssue), so it is functioning as a shared resource.

I try to avoid the pattern currently in the app (importing a utility or component from a sibling directory), because it's easy to mistakenly think that the hook defined within that feature is only used in that feature, and to accidentally break its uses elsewhere. In this case, this shared hook lives in the `More` feature, but is imported into other features very often.

I was thinking about this `hooks` directory kind of like our `components` directory-- Not all components go into it, but ones that are widely reused throughout the app do. Is there a more clear place for it?